common/app/conf/switches/CommercialSwitches.scala

```diff
@@ -295,6 +295,16 @@ trait CommercialSwitches {
     exposeClientSide = false
   )
 
+  val BusinessToBusinessNavLink: Switch = Switch(
```

Comment:
Out of curiosity, why would this be turned off? Is the switch worth it to have?

Reply:
The switch will probably be removed within the next couple of weeks. B2B is having a soft-launch, so it may be helpful to give the Labs team a bit more control if they want to hide the links for any reason 😊